src/savegame_json.cpp

```diff
@@ -2666,6 +2666,20 @@ void label::serialize( JsonOut &json ) const
     json.end_object();
 }
 
+void smart_controller_config::deserialize( const JsonObject &data )
+{
+    data.read( "bat_lo", battery_lo );
+    data.read( "bat_hi", battery_hi );
+}
```

Comment:
```suggestion
void smart_controller_config::deserialize( JsonIn &jsin )
{
    JsonObject data = jsin.get_object();
    data.read( "bat_lo", battery_lo );
    data.read( "bat_hi", battery_hi );
}
```
To make the method have the same signature as other `deserialize` methods.